azure-mgmt-resources/src/main/java/com/azure/management/resources/DeploymentProperties.java

```diff
@@ -24,7 +22,8 @@
     private Object template;
 
     /*
-     * Entity representing the reference to the template.
+     * The URI of the template. Use either the templateLink property or the

```

Comment:
is this change caused by generator change also?

Reply:
Yes. I think it is from changes by modelerfour. Not confirmed.